static/js/vue-cdr-access/src/mixins/fullRecordUtils.js

```diff
@@ -87,6 +89,35 @@ export default {
 
         toggleMetadata(show) {
             this.showMetadata = show;
+        },
+
+        fieldExists(value) {
+            return value !== undefined;
+        },
+
+        formatDate(value) {
+            return formatInTimeZone(value, 'America/New_York', 'yyyy-MM-dd');
+        },
+
+        editDescriptionUrl(id) {
+            return `https://${window.location.host}/admin/describe/${id}`;
+        },
+
+        hasGroupRole(permission, user_type= 'everyone') {
```

Comment:
I'd suggest not naming the first parameter here `permission`, its confusing since there is `hasPermission` directly below it. How about `role` or `expected_role`?